branding/css/theme.less

```diff
@@ -890,3 +890,58 @@ code {
     right: 0px;
   }
 }
+
+.multi-input-table-row {
+  display: flex;
+  .column-title {
+    font-weight: bolder;
+    margin-left: -15px;
+    margin-right: -15px;
+    padding-left: 30px;
+  }
+}
+
+// Represents network interfaces
+.interface {
```

Comment:
Here and below, I would recommend extracting these into a separate file and importing that in this file. These are fairly specific and this file is already pretty large on its own right.

Reply:
OK, I added the import in the global less files though, with the other imports.